src/pages/ApplicationDetail/ApplicationDetailSchoolPage.tsx

```
@@ -0,0 +1,177 @@
+import BaseHeader from '@/components/Common/Header/BaseHeader';
+import PageTitle from '@/components/Common/PageTitle';
+import {
+  useGetSchoolInfo,
+  usePatchContactCoordinator,
+} from '@/hooks/api/useApplication';
+import useNavigateBack from '@/hooks/useNavigateBack';
+import RecruiterIcon from '@/assets/icons/ApplicationDetail/RecruiterIcon.svg?react';
+import { Map, MapMarker } from 'react-kakao-maps-sdk';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)